data/processed/hesa/gbp_research_income.yaml

```diff
@@ -0,0 +1,22 @@
+api: the type of API to be used to fetch data: Download
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/estates/table-1
+description: Total amount of research income (GBP)
+endpoint_source: https://www.hesa.ac.uk/data-and-analysis/estates/data.csv
+framework_group: 1
+is_experimental: False
+nuts_level: 2
+source: the data provider: HESA (Higher Education Statistical Agency) https://www.hesa.ac.uk/
+year_range: ['2015/2016', '2017/18']
+schema:
+  nuts_id:
+    type: NutsRegion.id
+  nuts_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: Research income for universities in the region
+    id: gbp_research_income
+    label: Research income (GBP)
+    type: int
+  year:
+    data_type: str
+    label: Academic year
```

Comment:
`Start of the academic year` or `End of the academic year`? (we need to define or use a convention here)

Reply:
I have gone from Start. Saying "Level income on the academic year starting on..." feels easier to understand than level of income in the year ending on...

Easy to change if we decide to in any case.